packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -60,6 +66,69 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin[] {
 	let hasClientBuild = false;
 
 	return [
+		// Plugin that provides an __debug path for debugging the Cloudflare Workers.
```

Comment:
Can you try moving this plugin after the main plugin and then using the object form of `configureServer` and `configurePreviewServer` e.g.

```
configureServer: {
  order: 'pre',
  handler(viteDevServer) {
    // ...
  }
};
```

Should work but if it doesn't you can leave it as it is.

Reply:
Actually, you should just be able to set `enforce: 'pre'` on the plugin, no? Let me check.

Reply:
Yep, that works. So, yeah, if you can move the plugin after the main one and set `enfore: 'pre'` that would be great. Thanks!

Reply:
I don't really like the fact that `enforce: 'pre'` then put this before all userland plugins 😕 (moving it to the top of our array is a change only scoped to us), but it's not a huge deal, if that's your preference I'm ok with it 👍